packages/web/src/components/Alert/_theme.scss

```diff
@@ -0,0 +1,19 @@
+@use "sass:map";
+@use '@tokens' as tokens;
```

Comment:
Poor Stylelint… Could we have the same quotes here, no matter which? 🙂

Reply:
Fixed.